.github/workflows/pr-title.yaml

```diff
@@ -19,9 +19,9 @@ jobs:
       - uses: morrisoncole/pr-lint-action@v1.7.0
```

Comment:
Might be worth to also use the sha here, particularly if we'll roll it wide across our repos.